src/test/java/core/basesyntax/service/FileReaderImplTest.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.service;
+
+import core.basesyntax.service.impl.FileReaderImpl;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileReaderImplTest {
+    private static final String FILE_EXIST = "src/test/java/core/basesyntax/resources/input.csv";
+    private static final String WRONG_FILE_PATH = "src/test/wrongPath/input.csv";
+    private static FileReader fileReader;
+
+    @BeforeClass
+    public static void setUp() {
+        fileReader = new FileReaderImpl();
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void readFile_wrongPath_notOk() {

```

Comment:
```suggestion
    public void readFromFile_wrongPath_notOk() {
```
first part is the name of the method that you test, fix naming in every test method